Application/Services/UserService.cs

```diff
@@ -46,5 +48,19 @@ private static ClaimsIdentity GenerateClaims(UserResponse user)
 
             return claims;
         }
+
+        public async Task<LoginResponse> Login(LoginRequest request)
+        {
+            var userExist = await _userRepository.GetUserByAsync(u => u.EmailAddress == request.EmailAddress);
+            if (userExist != null && request.Password == userExist.PasswordHash)
```

Comment:
Can you be camparing request.Password with Hash?

Reply:
No sir, I was supposed to compare with the decrypted passwordhash 
But the password has not been hashed yet so I can't  decrypt it 
It would still be changed when the tool for hashing the password has been finalized sir

Reply:
Rather don't put a  code than put a wrong implementation.  That is a bad practice

Reply:
ok sir